emukit/core/parameter_space.py

```diff
@@ -37,7 +39,17 @@ def get_parameter_by_name(self, name):
         for param in self.parameters:
             if param.name == name:
                 return param
-        raise ValueError('Parameter with name ' + name + 'not found.')
+        raise ValueError('Parameter with name ' + name + ' not found.')
+
+    def round(self, x):
```

Comment:
+ Doc string
```suggestion
    def round(self, x: np.ndarray) -> np.ndarray:
```

Reply:
Added docstring there and to the rest of undocumented methods in this class